test/cassava/client_test.rb

```diff
@@ -161,6 +161,25 @@ def string_keys(hash)
       end
     end
 
+    context 'select_ttl' do
+      should 'allow an existing ttl to be read' do
+        ttl = 12345
+        item = { :id => 'i', :a => 1, :b => 'b', :c => "'\"item(", :d => 1, :ttl => ttl }
+        @client.insert(:test, item)
+
+        resulting_ttl = @client.select_ttl(:test, :d, {:id => 'i'})
+        assert (1..ttl).include? resulting_ttl
```

Comment:
Why the `include?` in this test instead of just an `assert_equal ttl, resulting_ttl`?